legal-api/src/legal_api/models/business.py

```diff
@@ -298,8 +298,7 @@ def get_ar_dates(self, next_ar_year):
             ar_min_date = datetime(next_ar_year, self.founding_date.month, self.founding_date.day).date()
             ar_max_date = ar_min_date + datedelta.datedelta(days=60)
 
-        if ar_max_date > datetime.utcnow().date():
-            ar_max_date = datetime.utcnow().date()
+        ar_max_date = min(ar_max_date, datetime.utcnow().date())  # ar_max_date cannot be in future
```

Comment:
A new pylint error says `Consider using 'ar_max_date = min(ar_max_date, datetime.utcnow().date())' instead of unnecessary if block (consider-using-min-builtin)`